### PR TITLE
Update Google static map options

### PR DESCRIPTION
**Pull request by tardinha**
* Update Google static map to use optional scale, as well as make height and width optional.
* Added Google API key to static map
* Removed depreciated sensor parameter

### CODE REVIEW

File: templates/AddressMap.ss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="addressMap">
 	<a href="//maps.google.com/?q=$Address">
-		<img src="//maps.googleapis.com/maps/api/staticmap?size={$Width}x{$Height}&markers=$Address&sensor=false" alt="$FullAddress.ATT" />
+		<img src="//maps.googleapis.com/maps/api/staticmap?size={$Width}x{$Height}&scale={$Scale}&maptype={$MapType}&markers=$Address&key=$Key" alt="$FullAddress.ATT" />
```
Comment (nglasl) [inline templates/AddressMap.ss]: Where does `$MapType` come from, and should there be an `if` statement around this? Also, with this and sensor removed, what version bump do you consider this having in relation to semantic versioning? Will this change break anything for users who are already using a static map from this module?

File: templates/AddressMap.ss [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <div class="addressMap">
 	<a href="//maps.google.com/?q=$Address">
-		<img src="//maps.googleapis.com/maps/api/staticmap?size={$Width}x{$Height}&markers=$Address&sensor=false" alt="$FullAddress.ATT" />
```
Comment (nglasl) [inline templates/AddressMap.ss]: My only remaining question with this is how removing `sensor` affects the map should someone already be using it? Other than that, this looks good to merge through.

File: templates/AddressMap.ss (hunk 1 above)
Comment (tardinha) [reply templates/AddressMap.ss]: I've removed $MapType.

Sensor has been removed: https://developers.google.com/maps/documentation/static-maps/intro#Moreinfo

I've tested this on existing sites and everything works correctly.

File: templates/AddressMap.ss (hunk 2 above)
Comment (nglasl) [reply templates/AddressMap.ss]: @tardinha, just following up to make sure I can get this merged through for you.

Comment (tardinha) [reply templates/AddressMap.ss]: @nglasl Sensor has been removed: https://developers.google.com/maps/documentation/static-maps/intro#Moreinfo :)

Comment (nglasl) [reply templates/AddressMap.ss]: Ah, well that makes sense then :)

**Comment by nglasl** [comment]
@tardinha, this has been merged and tagged 2.0.2.

**Pull request merged**